src/main/java/org/cardanofoundation/job/repository/explorer/jooq/JOOQBlockStatisticsPerEpochRepository.java

```diff
@@ -0,0 +1,154 @@
+package org.cardanofoundation.job.repository.explorer.jooq;
```

Comment:
Is there a reason why we do it with JOOQ and not with standard jpa?

Reply:
@Kammerlo with JOOQ, we can use the upsert feature with onConflict().doUpdate() in PostgreSQL. For standard JPA, we can also upsert records using the saveAll function. If you want to switch to JPA, we can do that.